molecular/src/test/kotlin/com/hartwig/actin/molecular/panel/PanelVariantAnnotatorTest.kt

```diff
@@ -135,23 +106,23 @@ private val PAVE_ANNOTATION = PaveResponse(
     transcriptImpact = emptyList()
 )
 
-private val HOTSPOT = TestServeKnownFactory.hotspotBuilder().build()
-    .withGeneRole(ServeGeneRole.ONCO)
-    .withProteinEffect(ServeProteinEffect.GAIN_OF_FUNCTION)
-    .withAssociatedWithDrugResistance(true)
+private val HOTSPOT_CKB =
+    TestVariantAlterationFactory.createVariantAlteration(GENE, GeneRole.ONCO, ProteinEffect.GAIN_OF_FUNCTION, true, true)
+
+private val HOTSPOT_DOCM = TestVariantAlterationFactory.createVariantAlteration(GENE, isHotspot = true)
```

Comment:
If not mistaken, these objects are not CKB or DoCM-related? 

Would rename to something like `HOTSPOT_1` and `HOTSPOT_2` or perhaps more descriptive names that at least don't mention something that isn't true!

In general, panel variant annotator doesn't make assumptions about origin and type of alterations anymore so would reflect that in the test (you may be able to get away with a single hotspot, though didn't check in detail)